MediaBrowser.Common/Net/IPHost.cs

```diff
@@ -394,15 +394,15 @@ protected override IPObject CalculateNetworkAddress()
         private bool ResolveHost()
         {
             // When was the last time we resolved?
-            if (_lastResolved == 0)
+            if (_lastResolved == null)
             {
-                _lastResolved = DateTime.UtcNow.Ticks;
+                _lastResolved = DateTime.UtcNow;
             }
 
             // If we haven't resolved before, or our timer has run out...
-            if ((_addresses.Length == 0 && !Resolved) || (TimeSpan.FromTicks(DateTime.UtcNow.Ticks - _lastResolved).TotalMinutes > Timeout))
+            if ((_addresses.Length == 0 && !Resolved) || (DateTime.UtcNow > _lastResolved?.AddTicks(Timeout)))
```

Comment:
1 tick is 1e7 ms. This will be resolved every time. You probably want to use AddMinutes